src/DataTable/tests/BulkActions.test.jsx

```diff
@@ -276,4 +278,13 @@ describe('<BulkActions />', () => {
       expect(icon.props().screenReaderText).toEqual(SMALL_SCREEN_DROPDOWN_BUTTON_TEXT);
     });
   });
+
+  describe('with function', () => {
```

Comment:
We could probably include test case(s) for the added `PropTypes.arrayOf(PropTypes.func)` arguments.